ci-operator/config/openshift/aws-load-balancer-operator/openshift-aws-load-balancer-operator-main.yaml

```diff
@@ -20,6 +24,17 @@ images:
       as:
       - golang:1.17
   to: aws-load-balancer-operator
+operator:
+  bundles:
+  - as: aws-load-balancer-operator-bundle
+    base_index: operator-index
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: openshift.io/aws-load-balancer-operator:latest
+    with: pipeline:aws-load-balancer-operator
+promotion:
```

Comment:
Do we need to promote the image?

Reply:
Yes, for some reason, the CI fails in the bundle-gen step without promotion. (not entirely sure why)

Reply:
Nvm, turns out it wasn't required.